pwndbg/aglib/disasm/arch.py

```diff
@@ -126,6 +126,21 @@ def syntax_highlight(ins):
 }
 
 
+def register_assign(left: str, right: str) -> str:
+    return f"{left} => {right}"
+
+
+def memory_assign(left: str, right: str) -> str:
+    return f"{left} <= {right}"
```

Comment:
we introduce lots of function calls with those and funccalls are expensive in py, but maybe it doesnt hurt

Reply:
I get that - worst case, these functions are called once per instruction shown on the screen.